src/lib/tabs/ink-bar.ts

```diff
@@ -38,6 +38,7 @@ export const _matInkBarPositioner: _MatInkBarPositioner = (element: HTMLElement)
 /**
  * The ink-bar is used to display and animate the line underneath the current active tab label.
  * @docs-private
+ * @dynamic
```

Comment:
I didn't find a way to make this one work without the `dynamic` flag. Will look at this later.

Reply:
I actually have no idea what this is. Is it documented somewhere?

Reply:
yes, it’s documented here:
https://angular.io/guide/aot-compiler#strictmetadataemit

The actual error is related to lambda error.

What’s is the reason that the positioner is an injector token, rather than
a private method within the component class?